framework/include/network/mqtt/mqtt_api.h

```diff
@@ -81,31 +81,35 @@ enum mqtt_connection_result_e {
 /**
  * @brief Structure of MQTT message
  */
-typedef struct _mqtt_msg_t {
+struct _mqtt_msg_t {
 	int msg_id;	/**< message identification */
 	char *topic; /**< message topic */
 	void *payload; /**< message payload content */
 	int payload_len; /**< message pay load length */
 	int qos; /**< message qos flag */
 	int retain;	/**< message retain flag */
-} mqtt_msg_t;
+};
+
+typedef struct _mqtt_msg_t mqtt_msg_t;
```

Comment:
Please check the below
struct XXX_s
typedef XXX_t

Reply:
fixed